src/DrugResponseModel.jl

```diff
@@ -16,6 +16,7 @@ import ExponentialUtilities
 import LineSearches
 using Polynomials
 using DiffResults
+using Interact
```

Comment:
If Interact isn't used within the package then it doesn't need to be imported or placed in the package manifest.

Reply:
Should I remove changes made to the manifest.toml, too?

Reply:
What is the purpose of the manifest?

Reply:
It has information about dependencies. So I'll remove the changes.